src/components/footer.js

```diff
@@ -19,7 +19,7 @@ module.exports = React.createClass({
           </div>
           <div className="horizontal-rule"></div>
           <p>
-            Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good. Visit us at <a href="//mozilla.org">mozilla.org</a>
+            Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the Web for the public good. Visit us at <a href="//mozilla.org">mozilla.org</a>
```

Comment:
Just some copy changes from feedback. Unrelated to the patch, but it was requested during review anyway.